packages/webamp-modern-2/src/skin/VM.ts

```diff
@@ -18,9 +19,11 @@ export default class Vm {
         ) {
           const reversedArgs = [...args].reverse();
           this.interpret(scriptId, binding.commandOffset, reversedArgs);
+          ran++;
         }
       }
     }
+    return ran
```

Comment:
Where do we need this?

Reply:
in Slider.ts.
it is needed to make double seeker works differently.
one of them is seekerGhost that should detect whether the maki already processed the slider.move.
seekerGhost is not a realtime slider, it is used to **show the position** of it's own into screen.
BUt not all skin has such seeker ghost. so we should preserve both behaviour: with/without realtime changing.

Reply:
seekerghost is also entirely optional and not mandatory in a skin, as can be witnessed in 0x5066/winamp1